core/src/main/java/io/kestra/core/tasks/debugs/Echo.java

```diff
@@ -37,23 +29,4 @@
     }
 )
 @Deprecated
-public class Echo extends Task implements RunnableTask<VoidOutput> {
-    @NonNull
-    @NotBlank
-    @PluginProperty(dynamic = true)
-    private String format;
-
-    @Builder.Default
-    @PluginProperty
-    private Level level = Level.INFO;
-
-    @Override
-    public VoidOutput run(RunContext runContext) throws Exception {
-        Log log = Log.builder()
-            .level(this.level)
-            .message(this.format)
-            .build();
-        log.run(runContext);
-        return null;
-    }
-}
+public class Echo extends Task {}
```

Comment:
You cannot remove all that code, this is breaking for our users (and it breaks every tests)